gemini/use-cases/retrieval-augmented-generation/rag_evaluation/ragas_with_gemini.ipynb

```diff
@@ -0,0 +1,816 @@
+      "\u001b[1;31mView Jupyter <a href='command:jupyter.viewOutput'>log</a> for further details."
+     ]
+    }
+   ],
+   "source": [
+    "import IPython\n",
+    "\n",
+    "app = IPython.Application.instance()\n",
+    "app.kernel.do_shutdown(True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "4TCvupe_UQvS"
+   },
+   "source": [
+    "After the restart is complete, continue to the next step.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "EXQZrM5hQeKb"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "<b>⚠️ Wait for the kernel to finish restarting before you continue. ⚠️</b>\n",
+    "</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "jXHfaVS66_01",
+    "tags": []
+   },
+   "source": [
+    "## Import libraries\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "executionInfo": {
+     "elapsed": 15245,
+     "status": "ok",
+     "timestamp": 1708026827793,
+     "user": {
+      "displayName": "",
+      "userId": ""
+     },
+     "user_tz": 300
+    },
+    "id": "lslYAvw37JGQ",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import vertexai\n",
+    "\n",
+    "from langchain_community.document_loaders import PyPDFLoader\n",
+    "from langchain_chroma import Chroma\n",
+    "from langchain.chains import RetrievalQA\n",
+    "from langchain.prompts import PromptTemplate\n",
+    "\n",
+    "from langchain_google_vertexai import VertexAI, VertexAIEmbeddings\n",
+    "\n",
+    "# Important to make Gemini Work with RAGAS\n",
+    "from ragas.llms.base import LangchainLLMWrapper\n",
+    "from ragas import evaluate\n",
+    "from ragas.metrics import (\n",
+    "    answer_relevancy,\n",
+    "    context_recall,\n",
+    "    context_precision,\n",
+    ")\n",
+    "from ragas.metrics.critique import harmfulness\n",
+    "\n",
+    "from datasets import Dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# TODO(developer): Update the below lines\n",
+    "PROJECT_ID = \"<your_project>\"\n",
+    "LOCATION = \"<your_region>\"\n",
+    "\n",
+    "vertexai.init(project=PROJECT_ID, location=LOCATION)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "4437b7608c8e"
+   },
+   "source": [
+    "## Use Vertex AI models\n",
+    "\n",
+    "The [Gemini 1.5 Pro](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/overview#text-use-cases) models are designed to handle natural language tasks, multiturn text and code chat, and code generation.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load the Gemini Pro model\n",
+    "llm = VertexAI(model_name=\"gemini-1.0-pro-002\")"
```

Comment:
Earlier, you mentioned using 1.5 Pro, but this is for 1.0

Reply:
@Adi8885 Please resolve this. Be consistent in model usage, unless you are intentionally using a different model.

Reply:
Done